source/develop/release-management/features/virt/KvmToOvirt.html.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Importing Libvirt KVM VMs to oVirt
+category: feature
+authors: Shahar Havivi
+wiki_category: Feature|v2v
+wiki_title: Features/KvmToOvirt
+wiki_revision_count: 1
+wiki_last_updated: 2017-03-20
+feature_name: 'v2v: Importing Libvirt Kvm Vms to oVirt'
+feature_modules: all
+feature_status: Released in oVirt 4.1
+---
+
+# Importing Libvirt KVM VMs to oVirt
+
+## Summary
+Import KVM based VMs from Libvirt management via the oVirt admin portal.
+There is no need for disk conversion since both oVirt and Libvirt support KVM based VMs.
+Importing VMs with file and block device disks is supported, snapshots are currently not supported. Please note that if you want to import VM with snapshots the current snapshot is the one that will be imported.
+The import process is done via the utility `kvm2ovirt` that comes in VDSM installation. It is located at `/usr/libexec/vdsm/kvm2ovirt`.
+
+## Requirements on VMs
+
+* The VMs that you want to import must be in powered off.
+
+* It is expected that the VirtIO drivers are already installed in the VM. While this is usually already the case for Linux VMs, special care has to be taken for Windows VMs.
+The drivers are not installed during the import (as it is for VMware/Xen VMs imported via virt-v2v).
+If the drivers are missing the VM will probably not be able to boot after the import. Alternatively, review the configuration of the VM after the import and make sure VirtIO devices are not used. Bear in mind that using emulated devices has performance penalties.
+
+## Libvirt Configuration
+In order to import VM from Libvirt, Libvirt needs to listen on a public port or be reachable over SSH.
+You can read the respective documentation on how to setup Libvirt for connections
+[over SSH](https://wiki.libvirt.org/page/SSHSetup),
+[over TCP](https://wiki.libvirt.org/page/Libvirt_daemon_is_not_listening_on_tcp_ports_although_configured_to), or
+[over TLS](https://wiki.libvirt.org/page/TLSSetup).
+The URI for accessing Libvirt then depends on the method you have chosen.
+For access over SSH use `qemu+ssh://username@host1.example.org/system`, for TCP `qemu+tcp://username@host1.example.org/system`
+and for TLS connection use `qemu+tls://username@host1.example.org/system`.
+
+## Importing VM
+Importing VM is done by the VM unique Libvirt name that can be identified via:
+
+    $ virsh -r -c 'qemu+tcp://username@host1.example.org/system' list --all
+
+For example if we want to import a VM named `rhel1_local` from `qemu+tcp://username@host1.example.org/system`:
+
+- Login to the *Admin Portal* and navigate to the *Virtual Machines* tab
+- Press the *Import* button on the toolbar ![](/images/wiki/ImportFromKvm1.png)
+- In the Source box select *KVM (via libvirt)* option ![](/images/wiki/ImportFromKvm2.png)
+- Enter `qemu+tcp://username@host1.example.org/system` in the URI box
+- Press the *Requires Authentication* checkbox if authentication is needed and enter username/password
```

Comment:
Suggest:
"If authentication is required, check the *Requires Authentication* checkbox, and enter username/password"